js/ui/handler/touch_zoom_rotate.js

```diff
@@ -41,13 +41,23 @@ class TouchZoomRotateHandler {
     /**
      * Enables the "pinch to rotate and zoom" interaction.
      *
+     * @param {Object} options
+     * @param {string} options.around If "center" is passed, map will zoom around the center
+     *
      * @example
      *   map.touchZoomRotate.enable();
+     * @example
+     *   map.touchZoomRotate.enable({ around: 'center' });
      */
-    enable() {
+    enable(options = {}) {
```

Comment:
Our [code conventions](https://github.com/mapbox/mapbox-gl-js/blob/master/CONTRIBUTING.md#code-conventions) don't currently permit the use of default parameters.